jjb/edgex-macros.yaml

```diff
@@ -34,3 +34,25 @@
       - shell: !include-raw-escape:
           - global-jjb/shell/lftools-install.sh
           - shell/edgex-publish-docs.sh
+
+- builder:
+    name: edgex-provide-snap-cleanup
+    builders:
+      - shell: |
+          #!/bin/bash
+          set +e  # DO NOT cause build failure if any of the rm calls fail.
+
+          rm -f "$HOME/EdgeX"
+
+          # DO NOT fail build if any of the above lines fail.
+          exit 0
```

Comment:
You shouldn't be inlining shell scripts. Doing so makes it impossible to run ShellCheck against the code.